pages/cartPage/cartPage.css

```diff
@@ -0,0 +1,54 @@
+@import "/styles.css";
+
+.page-wrap {
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_footer p_footer p_footer p_footer p_footer p_footer';
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+/* TODO: Needs attention to get fixed */
+.dui-card {
+    width: 4rem;
+}
+
+.main-content__cart-container {
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 3rem;
+}
+
+.dui-card:not(:last-child) {
+    margin-bottom: 1.6rem;
+}
+
+.dui-cart-details-card {
+
+    width: fit-content;
+    min-width: 50rem;
+    height: fit-content;
+    padding: 2rem;
+    padding-left: 7.5rem;
+    padding-right: 7.5rem;
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    background-color: white;
```

Comment:
Use var for colours

Reply:
Addressed the comment in the follow-up:
https://github.com/varun13169/gardenkart/pull/8